kong/plugins/zipkin/schema.lua

```diff
@@ -61,6 +61,7 @@ return {
           { tags_header = { type = "string", required = true, default = "Zipkin-Tags" } },
           { static_tags = { type = "array", elements = static_tag,
                             custom_validator = validate_static_tags } },
+          { span_name_include_path = { type = "boolean", default = false } },
```

Comment:
Hi, I propose the following change to the field:

```
          { http_span_name = { type = "string", required = true, default = "method", one_of = { "method", "method_path" } } },
```

Reasons for the changes:
* A string (enum) instead of a boolean: With a boolean we "lock" ourselves to two options. With an enum we leave the door open to other span names in the future. The field name had to be changed to reflect this as well.
* `http_` prefix: The path will only be available on http requests. I think that detail is fundamental enough to be part of the field name.

Reply:
Thanks! Using the enum type in the schema is more flexible and extensible.